### PR TITLE
SSR 6.0.7 release notes

### PR DESCRIPTION
**Pull request by Chr1st0ph3rTurn3r**

### CODE REVIEW

File: docs/releases.table.js [hunk 1]
```diff
@@ -5,7 +5,14 @@ import React from "react";
 // New entries should be placed at the top.
 // ----------------------------------------------------------------------------
 export const releases = [
-    {
+  {
+    version: '6.0.6',
+    url: 'release_notes_128t_6.0#release-606-9',
+    releaseDate: 'November 13, 2022',
```
Comment (MichaelBaj) [inline docs/releases.table.js:11]: Target date needs to be updated with actual.

File: docs/release_notes_128t_6.0.md [hunk 2]
```diff
@@ -24,6 +24,85 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
 ------
 - **Plugin Upgrades:** If you are running with plugins, updates are required for some plugins **before** upgrading the conductor to SSR version 5.4.0 or higher. Please review the [Plugin Configuration Generation Changes](intro_upgrade_considerations.md#plugin-configuration-generation-changes) for additional information.
 
+## Release 6.0.6-9
+
+**Release Date:** November 13, 2022
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.0.md:29]: To be updated with actual

File: docs/release_notes_128t_6.0.md [hunk 3]
```diff
@@ -24,6 +24,85 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+	- overrideServiceName
+	- appStatsTrackingKey (combination of application, client ip, ingress-interface, next-hop, and traffic-class)
+
+### Resolved Issues
+
+- **I95-32789 Peer metrics unavailable after Conflux synchronization:** Resolved an issue with HA routers where the metrics application stops streaming metrics to the peer node after loading configuration.
+------
+- **I95-43302 Rename Third-Party menu text:** The menu text has been changed to **External** to more accurately reflect the links to other Juniper platforms.
+------
+- **I95-44957 Azure is not able to identify the asset-id of the depolyed conductor and router:** The Azure ID has been modified to a value that can be processed by Azure.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.0.md:46]: typo: deployed

File: docs/release_notes_128t_6.0.md [hunk 4]
```diff
@@ -24,6 +24,85 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-48246 Peer path GQL query should provide a node filter:** Added a parameter to stats on peer-path so that the node can be overwritten.
+------
+- **I95-48353 Optimize Application-ID stats allocation:** Reduced memory usage by avoiding unnecessary app-id stats tracking allocations.
+------
+- **I95-48357 CoreDump on Failover with DSCP Steering:** Resolved an issue where DSCP Steering sessions would fail to move a flow under certain circumstances and, when using DSCP value 0, crash.
+------
+- **I95-483381 Race condition in session teardown:** Shared context is now maintained to allow all packet processing to be completed before session teardown.
+------
+- **I95-48427 BGP ignoring multihop ttl setting leading to invalid nexthop:** Resolved an issue where BGP may temporarily "forget" about the ttl value configured for a neighbor.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.0.md:100]: ttl should be capitalized (Time To Live)

File: docs/release_notes_128t_6.0.md [hunk 5]
```diff
@@ -24,6 +24,85 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-48357 CoreDump on Failover with DSCP Steering:** Resolved an issue where DSCP Steering sessions would fail to move a flow under certain circumstances and, when using DSCP value 0, crash.
+------
+- **I95-483381 Race condition in session teardown:** Shared context is now maintained to allow all packet processing to be completed before session teardown.
+------
+- **I95-48427 BGP ignoring multihop ttl setting leading to invalid nexthop:** Resolved an issue where BGP may temporarily "forget" about the ttl value configured for a neighbor. 
+------
+- **I95-48476 SSR cannot be adopted into Mist if DNS server 1.1.1.1 is unreachable:** Resolved an issue where blocking 1.1.1.1 prevented the `adopt` command and the GUI first-time mist onboarding process from completing. 
+------
+- **I95-48507 Vlan packets are generated without a valid vlan from the flow-move cache:** Resolved an issue where sessions could be modified incorrectly when vlan is present when session resiliency is enabled for failover.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.0.md:104]: VLAN should be capitalized

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/release_notes_128t_6.0.md [hunk 6]
```diff
@@ -24,6 +24,85 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-47075 Disable weak SSH ciphers:** Resolved issues where the remote SSH server was configured to allow weak key exchange algorithms on `tcp/22` and `tcp/930`. 
+------
+- **I95-47271 VRRP Alarm for Backup becoming Primary:** There is now an alarm when the backup VRRP node in an HA pair takes over as the primary.
+------
+- **I95-47475 Session capture not downloadable for a read only user:** Adjusted permissions to provide access to session capture files to read-only users.
+------
+- **I95-47476 Session table associated paths not scalable, scroll bar hidden:** The Session Table window has been enlarged to more clearly show information.
+------
+- **I95-47519 Add First Article of Inspection scan output to Tech Support Info:** Added the output of the FAI (first article of inspection) script to Tech Support Info.
```
Comment (ptimmons) [inline docs/release_notes_128t_6.0.md:64]: FAI = first article inspection (no "of")

File: docs/releases.table.js [hunk 7]
```diff
@@ -6,12 +6,33 @@ import React from "react";
 // ----------------------------------------------------------------------------
 export const releases = [
   {
-    version: '6.0.6',
-    url: 'release_notes_128t_6.0#release-606-9',
-    releaseDate: 'November 13, 2022',
+    version: '6.0.7',
+    url: 'release_notes_128t_6.0#release-607-6',
```
Comment (MichaelBaj) [inline docs/releases.table.js:10]: looks like you will need to rebase from master. These URLs should all have the prefix `/docs/` on them.

File: docs/release_notes_128t_6.0.md [hunk 8]
```diff
@@ -61,13 +70,13 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
 ------
 - **I95-47787 Worker core packet processing spikes to 100%:** Added the ability to tune the [Reverse Packet Session Resiliency](config_reference_guide.md#reverse-packet-session-resiliency) `Minimum Packet Count` (default is 3) and `Detection Interval` (default is 5) settings for session failover without requiring forward packet, and resolved the underlying issue that caused excessively high worker-core CPU.
 ------
 - **I95-47909 Handle GRE tunnels in ICMP reachability probe:** The base interface for egress is now used if the `icmp-probe probe-address` is the same as the tunnel destination, and the `internal-address` is used as the source if the `egress-interface` is `gre-overlay`.
 ------
-- **I95-47967 Cloud bootstrapper does not bootstrap the deployed Conductor:** Resolved an issue where the configuration was being rejected by the cloud bootstrapper when the device was a conductor.
+- **I95-47929 Missing BGP advertisement after deleting all sessions after an upgrade:** Resolved an issue where FRR BGP update suppress was not removing any pending withdrawls.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.0.md:79]: Simply remove `FRR` here.

File: docs/release_notes_128t_6.0.md [hunk 9]
```diff
@@ -91,20 +98,61 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
 ------
 - **I95-48246 Peer path GQL query should provide a node filter:** Added a parameter to stats on peer-path so that the node can be overwritten.
 ------
-- **I95-48353 Optimize Application-ID stats allocation:** Reduced memory usage by avoiding unnecessary app-id stats tracking allocations.
+- **I95-48343 Cannot complete Image-based installation if secondary disks are less than 2TB:** Resolved the issue so that image-based installation will complete correctly for systems containing drives larger than 2TB.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.0.md:101]: This statement seems contradictory. cannot complete when <2TB. Can complete when > 2TB

File: docs/release_notes_128t_6.0.md [hunk 10]
```diff
@@ -91,20 +98,61 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+- **I95-48507 VLAN packets are generated without a valid VLAN from the flow-move cache:** Resolved an issue where sessions could be modified incorrectly when a VLAN is present and session resiliency is enabled for failover.
+------
+- **I95-48508 Keep-alive cache may cause worker core CPU spikes:** Resolved potential worker core utilization CPU spikes by utilizing aggressive keep-alive timeouts.
+------
+- **I95-48529 BFD sending link notification before hold-down timer expires:** Resolved an issue where peer service-paths do not remain down while the BFD session / peer status is in the hold-down period after transitioning from down to up. Peer service-paths status now correctly reflect the peer status. Sessions will not be moved back to peers that have re-established connectivity but are still in the hold-down period.
+------
+- **I95-48579 Application director does not handle overlapping prefixes correctly:** The IP tree has been updated and now handles overlapping prefixes correctly.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.0.md:117]: "IP tree"? Do you mean "[trie](https://en.wikipedia.org/wiki/Trie)"? I would double check on this.

File: docs/release_notes_128t_6.0.md [hunk 11]
```diff
@@ -91,20 +98,61 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-48508 Keep-alive cache may cause worker core CPU spikes:** Resolved potential worker core utilization CPU spikes by utilizing aggressive keep-alive timeouts.
+------
+- **I95-48529 BFD sending link notification before hold-down timer expires:** Resolved an issue where peer service-paths do not remain down while the BFD session / peer status is in the hold-down period after transitioning from down to up. Peer service-paths status now correctly reflect the peer status. Sessions will not be moved back to peers that have re-established connectivity but are still in the hold-down period.
+------
+- **I95-48579 Application director does not handle overlapping prefixes correctly:** The IP tree has been updated and now handles overlapping prefixes correctly. 
+------
+- **I95-48580 Application summary classification fails for hub-to-spoke sessions:** The spoke now learns application names for sessions when receiving packets from a hub with application identification disabled.
+------
+- **I95-48581 No entry timestamp for s`how app-id cache`:** Additional timing information has been added to `show app-id cache`.
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.0.md:121]: `s` is outside of backquote

File: docs/release_notes_128t_6.0.md [hunk 12]
```diff
@@ -91,20 +98,61 @@ Before upgrading please review the [**Upgrade Considerations**](intro_upgrade_co
...
+------
+- **I95-48897 Adaptive encryption breaks after flow move:** Resolved an issue where the session breaks during failover when adaptive encryption is enabled.
+------
+- **I95-48904 Stuck pinhole session after flow invalidation:** Resolved an issue with a stuck session that was setup from hub to HA spoke after a routing change.
+------
+- **WAN-1372 Improve CPU Usage Reporting:** Devised a more efficient collection scheme to minimize the CPU impact when collecting the CPU and memory data.
 
 ## Release 6.0.5-17
 
+:::important
```
Comment (MichaelBaj) [inline docs/release_notes_128t_6.0.md:145]: Can we call out which issue ID relates to this problem and/or call out the release(s) in which this issue is addressed?

**Review by MichaelBaj — CHANGES_REQUESTED** [review]

File: docs/releases.table.js (hunk 7 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/releases.table.js:10]: Not sure what you mean. The same file in `master` looks the same.

File: docs/release_notes_128t_6.0.md (hunk 10 above)
Comment (Chr1st0ph3rTurn3r) [reply docs/release_notes_128t_6.0.md:117]: James said it is technically a "radix tree", so I changed it to that.

**Review by MichaelBaj — DISMISSED** [review]

**Review by jeffmart-jnpr — APPROVED** [review]

**Review by MichaelBaj — APPROVED** [review]

**Pull request merged**